File: StaticFramework/SFSample/SFSample/ViewController.m
//
//  ViewController.m
//  SFSample
//
//  Created by Hari Kunwar on 2/25/16.
//  Copyright © 2016 Learning. All rights reserved.
//

#import "ViewController.h"
#import <StaticFramework/SFCustomView.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    SFCustomView *view = [SFCustomView new];
    view.frame = CGRectMake(0, 0, 100, 100);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: MobileAds/MobileAdsSample/ViewController.swift
//
//  ViewController.swift
//  MobileAdsSample
//
//  Created by Hari Kunwar on 2/8/16.
//  Copyright © 2016 Learning. All rights reserved.
//

import UIKit
import MobileAds

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let _ = BannerAdView(frame: CGRectMake(0, 0, 300, 10))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}

